src/annotationLayer.js

```diff
@@ -351,6 +365,224 @@ var annotationLayer = function (args) {
     return this;
   };
 
+  /**
+   * Return the current set of annotations as a geojson object.  Alternately,
+   * add a set of annotations from a geojson object.
+   *
+   * @param {object} geojson: if present, add annotations based on the given
+   *    geojson object.  If undefined, return the current annotations as
+   *    geojson.  This may be a JSON string, a javascript object, or a File
+   *    object.
+   * @param {boolean} clear: if true, when adding annotations, first remove all
+   *    existing objects.  If 'update', update existing annotations and remove
+   *    annotations that no longer exit,  If false, update existing
+   *    annotations and leave unchanged annotations.
+   * @param {string|geo.transform} [gcs] undefined to use the interface gcs,
+   *    null to use the map gcs, or any other transform.
+   * @param {boolean} includeCrs: if true, include the coordinate system in the
```

Comment:
In geojson it is called `crs`.  Since this affects the geojson output, I figured it should be crs in this context, not gcs.